src/Fabulous.XamarinForms/Views/Controls/Image.fs

```diff
@@ -33,3 +48,17 @@ module ImageBuilders =
 
         static member inline Image<'msg>(stream: System.IO.Stream, aspect: Aspect) =
             View.Image<'msg>(ImageSource.FromStream(fun () -> stream), aspect)
+
+[<Extension>]
+type ImageModifiers =
+    [<Extension>]
+    static member inline aspect(this: WidgetBuilder<'msg, #IImage>, aspect: Aspect) =
+        this.AddScalar(Image.Aspect.WithValue(aspect))
+
+    [<Extension>]
+    static member inline isAnimationPlaying(this: WidgetBuilder<'msg, #IImage>, isAnimationPlaying: bool) =
+        this.AddScalar(Image.IsAnimationPlaying.WithValue(isAnimationPlaying))
```

Comment:
What's the usage for this property?

Reply:
The disc does not says https://docs.microsoft.com/en-us/dotnet/api/xamarin.forms.image.isanimationplayingproperty?view=xamarin-forms

Reply:
Might be for some internal special case  but is part of the specs .

Reply:
According to posts on StackOverflow, it seems to be for gif animation.